internal/app/components/courses/requirements_test.go

```diff
@@ -8,36 +8,31 @@ import (
 
 func TestCourseRequirement_IsSatisfied(t *testing.T) {
 	records := initRecords()
-	// testing using future course record (record3 )
-	courseId := CourseId(789)
+	// testing using future course record
 	req := CourseRequirement{
 		MinGrade: 100, // should ignore this requirement
-		CourseId: courseId,
+		CourseId: CourseId(789),
 	}
 	assert.Equal(t, true, req.IsSatisfied(records), "Test future record")
 
 	// missing requirement
-	courseId = CourseId(0)
 	req = CourseRequirement{
-		MinGrade: 0, // should ignore this requirement
```

Comment:
lmao thanks for cleaning up my copy pasta

Reply:
gotchu fam